ToGather/ToGather/Global/Service/OnboardingViewModel.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  LastOnboardingViewModel.swift
+//  ToGather
+//
+//  Created by Seungwon Choi on 2022/06/09.
+//
+
+import Foundation
+import Firebase
+import UIKit
+import SwiftUI
+
+class OnboardingViewModel: ObservableObject {
+    @Published var savingData = Saving(goalProduct: Product(productName: "", productPrice: 0, imageUrl: ""),
+                                       uid: "", goalWeeks: 0, startDate: Timestamp(date: Date()), savingDayOfTheWeek: "",
+                                       weekInfo: [ThisWeek(presentWeek: 0, didSave: false)])
+    @Published var userData = User(id: "", nickname: "", creationDate: Timestamp(date: Date()), isAlarmOn: true)
+    
+    // goal-setting 뷰에서 사용, savingData 인스턴스에 목표 product 추가
+    func addProduct(product: Product) {
+        savingData.goalProduct = product
+    }
+    
+    // setting-period 뷰에서 사용, savingData 인스턴스에 총 주차, 요일 추가
+    func addGoalWeekAndDayOfTheWeek(goalWeeks: Int, dayOfTheWeek: String) {
+        savingData.goalWeeks = goalWeeks
+        savingData.savingDayOfTheWeek = dayOfTheWeek
+    }
+    
+    // user 컬렉션에서 friend의 uid로 검색하여 친구 정보 가져오기
+    func getFriendInfo(friendUids: [String]){
+        let db = Firestore.firestore()
```

Comment:
타인이 이해하기 힘든 축약을 사용하기보다는 그대로의 이름이 좋답니다 : )
```suggestion
        let firestore = Firestore.firestore()
```